dashboard-config_postgres.yaml

```diff
@@ -0,0 +1,20 @@
+  dashboard:
```

Comment:
@ahetawal-p you'll probably want to delete this and the sqlite equivalent below. It'd be cool to add examples of the new `database` syntax to the README, minus your username. :)